eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -313,4 +345,80 @@ public function extractFilesFromXml($xml)
 
         return null;
     }
+
+    public function getAllImagesData(): array
+    {
+        $selectQuery = $this->connection->createQueryBuilder();
+        $selectQuery
+            ->select(
+                $this->connection->quoteIdentifier('contentobject_attribute_id'),
+                $this->connection->quoteIdentifier('filepath')
+            )->from($this->connection->quoteIdentifier(self::IMAGE_FILE_TABLE))
+            ->distinct()
```

Comment:
```suggestion
        $selectQuery = $this->connection->createQueryBuilder();
        $selectQuery
            ->select(
                $this->connection->quoteIdentifier('contentobject_attribute_id'),
                $this->connection->quoteIdentifier('filepath')
            )
            ->distinct()
            ->from($this->connection->quoteIdentifier(self::IMAGE_FILE_TABLE))
```
Nitpick: readablity.